### PR TITLE
add polish translation

### PR DESCRIPTION
**Pull request by burasuk**

### CODE REVIEW

**Comment by hanzei** [comment]
Hey @burasuk,

Thanks for the PR. The translation file looks badly formatted. Did you follow the steps at https://github.com/matterpoll/matterpoll/blob/master/CONTRIBUTING.md#translation-process-for-new-languages?

File: assets/i18n/active.pl.json [hunk 1]
```diff
@@ -0,0 +1,137 @@
...
+    "poll.button.deletePoll": {
+        "hash": "sha1-390ff4fdd0cfef6ef35fbf067a605aa3ea87ecca",
+        "other": "Usuń sondę"
+    },
+    "poll.button.endPoll": {
+        "hash": "sha1-60ddde386bd7190fa3da2ba13f97e3caa625d3e6",
+        "other": "Zakończ sondę"
+    },
+    "poll.endPost.answer.heading": {
+        "few": "{{.Answer}} ({{.Count}} głosów)",
```
Comment (hanzei) [inline assets/i18n/active.pl.json]: Are the additional translations for `few` and `many` really needed? I see no difference to `other`.

Comment (burasuk) [reply assets/i18n/active.pl.json]: > Thanks for fixing the formatting. Would you also be open on maintaining the polish translation in the future?

sure

> Are the additional translations for few and many really needed? I see no difference to other.

corrected

**Comment by hanzei** [comment]
Awesome, thanks @burasuk :tada:

**Pull request merged**